databases/week2/1and3part.sql

```diff
@@ -0,0 +1,66 @@
+use hyf_lesson2;
+SELECT *
+FROM task;
+SELECT*
+from status;
+
+--part 1
+
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES("shopping", 'by a car', now(), now(),now(),'2');
+
+--Change the title of a task
+UPDATE task
+SET title = 'cancel shopping'
+WHERE id = 36;
+
+--Change a task due date
+update task
+SET due_date = '2022-02-10'
+WHERE id = 36;
+
+--Change a task status
+update task
+SET status_id = '2'
+WHERE id = 36;
+
+--Mark a task as complete
+update task
+SET status_id = '3'
+WHERE id = 36;
```

Comment:
its correct, but, is it possible to do this in another way? lets say I know there is a status called 'Done', is it possible to update it to 'Done' without knowing what is the id of the corresponding status in this case 'Done' ?